src/index.ts

```diff
@@ -1254,7 +1254,7 @@ export class ValidationService {
      * @param removeClass Class to remove
      */
     private swapClasses(element: Element, addClass: string, removeClass: string) {
-        if (!element.classList.contains(addClass)) {
+        if (addClass && !element.classList.contains(addClass)) {
             element.classList.add(addClass);
         }
         if (element.classList.contains(removeClass)) {
```

Comment:
Does this throw if `removeClass` is blank? I wonder if we should protect against that too.

Reply:
Yes, it throws if you call `element.classList.remove('')` but the `contains(removeClass)` check prevents that from happening so I didn't add it for that case.

Reply:
Ah, makes sense. Thanks for following up and the contribution!